services/data_plane_manager/src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,14 @@
+#server.port=8090
```

Comment:
we could remove application-*.properties for now.

Reply:
changed in 9a10de4